src/handlers/localstorage/globalSettings.js

```diff
@@ -28,3 +29,8 @@ export const getNativeCurrency = () => getGlobal(NATIVE_CURRENCY, 'USD');
 
 export const saveNativeCurrency = nativeCurrency =>
   saveGlobal(NATIVE_CURRENCY, nativeCurrency);
+
+export const getImageMetadata = () => getGlobal(IMAGE_METADATA);
```

Comment:
Default for getGlobal is []. In my separate PR, I set the default value to getGlobal(IMAGE_METADATA, {}) to be more explicit.